pkg/v1/tkg/client/upgrade_cluster.go

```diff
@@ -435,6 +435,12 @@ func (c *TkgClient) applyPatchAndWait(regionalClusterClient, currentClusterClien
 	}
 
 	log.Info("Upgrading control plane nodes...")
+	log.Infof("Attempting to increase kube-vip timeouts")
+	err = c.increaseKubeVipTimeouts(regionalClusterClient, upgradeClusterConfig)
+	if err != nil {
+		log.Infof("Unable to modify kube-vip timeouts. Continuing to upgrade cluster with old kube-vip timeouts")
```

Comment:
```suggestion
		log.Warningf("Unable to modify kube-vip timeouts. Continuing to upgrade cluster with old kube-vip timeouts")
```
A warning or error sounds more appropriate.